### PR TITLE
start mbp setup instructions

### PR DESCRIPTION
**Pull request by ateucher**
WIP. @stephhazlitt @BonnieJRobert feel free to add to this branch!

### CODE REVIEW

**Comment by ateucher** [comment]
@stephhazlitt @BonnieJRobert I think this is good to go - let me know if you have anything else to add, or if I can merge?

**Review by stephhazlitt — APPROVED** [review]
Looks great, thanks so much for the effort with this.

File: docs/Macbook-pro-setup.md [hunk 1]
```diff
@@ -0,0 +1,231 @@
+# MacBook Pro Set Up 
+
+This document contains instructions for setting up a MacBook Pro in B.C. Government, tailored for data science workflows used by the Data Science Partnerships team.  The document is organized with least complex steps at the top (i.e. point and click downloads) leading to tools that require more advanced setup at the bottom. Some of the setup instructions assume a working knowledge of Unix/Linux.
```
Comment (BonnieJRobert) [inline docs/Macbook-pro-setup.md:3]: The second sentence may need revision as doc has been split.

**Review by BonnieJRobert — APPROVED** [review]
I haven't worked through the Advanced file so no comments on it at the moment.

File: docs/Macbook-pro-setup.md (hunk 1 above)
Comment (ateucher) [reply docs/Macbook-pro-setup.md:3]:
```suggestion
This document contains instructions for setting up a MacBook Pro in B.C. Government, tailored for data science workflows used by the Data Science Partnerships team.  The document is organized with least complex steps at the top (i.e. point and click downloads) leading to tools that require more advanced setup at the bottom. There is an [additional document](Macbook-advanced-geo.md) that contains some lesser-used advanced setup instructions as well as tips for setting up a geospatial data science environment. Some of the setup instructions assume a working knowledge of Unix/Linux command-line environments. 
```

**Comment by ateucher** [comment]
Thanks! Merging now.

**Pull request merged**